src/main/java/net/sf/marineapi/nmea/parser/SentenceFactory.java

```diff
@@ -278,6 +278,10 @@ public static SentenceFactory getInstance() {
 	/**
 	 * Resets the factory in it's initial state, i.e. restores and removes all
 	 * parsers the have been either removed or added.
+	 * 
+	 * Add new parsers for Boreal GasFinder DTA and DTB
```

Comment:
Changes should be documented in the [changelog.txt](https://github.com/ktuukkan/marine-api/blob/master/changelog.txt), not in actual API docs.

Reply:
Done